charts/aws-mountpoint-s3-csi-driver/templates/csidriver.yaml

```diff
@@ -12,4 +12,6 @@ spec:
   tokenRequests:
     - audience: "sts.amazonaws.com"
       expirationSeconds: 3600
+    - audience: "pods.eks.amazonaws.com"
+      expirationSeconds: 3600
```

Comment:
Should we wrap it with `{{- if .Values.experimental.podMounter -}}`?

Similar also in `provideFromPod()` method below with `usePodMounter`